admin-dev/themes/default/css/bundle/module/module.css

```diff
@@ -5,6 +5,10 @@
         height:50px;
 }
 
+.btn.onclic {
```

Comment:
nope, this is really `.onclic` in the UI-kit 

https://github.com/PrestaShop/prestashop-ui-kit/blob/4e7cb7e56c0644b94a0f4af9f11c0ae907e66301/scss/modules/_spinner.scss#L132-L163